api/coordinator.go

```diff
@@ -8,22 +8,15 @@ import (
 	"fmt"
 	"net/http"
 
+	client "github.com/mattermost/mattermost-load-test-ng/api/client/coordinator"
 	"github.com/mattermost/mattermost-load-test-ng/coordinator"
 	"github.com/mattermost/mattermost-load-test-ng/defaults"
 	"github.com/mattermost/mattermost-load-test-ng/loadtest"
 
 	"github.com/gorilla/mux"
 )
 
-// CoordinatorResponse contains the data returned by load-test coordinator API.
-type CoordinatorResponse struct {
-	Id      string              `json:"id,omitempty"`      // The load-test coordinator unique identifier.
-	Message string              `json:"message,omitempty"` // Message contains information about the response.
-	Status  *coordinator.Status `json:"status,omitempty"`  // Status contains the current status of the coordinator.
-	Error   string              `json:"error,omitempty"`   // Error is set if there was an error during the operation.
-}
-
-func writeCoordinatorResponse(w http.ResponseWriter, status int, resp *CoordinatorResponse) {
+func writecoordinatorResponse(w http.ResponseWriter, status int, resp *client.CoordinatorResponse) {
```

Comment:
Any reason the C became lower-case?

Reply:
I am guessing an erroneous string replace when I exported `CoordinatorResponse` struct.  
Nice catch :+1: